src/main/ml-modules/root/config/similarConfig.mjs

```diff
@@ -87,13 +83,9 @@ const SIMILAR_CONFIG = {
         child: null,
         paths: [
           // this was the field agentDescriptionNoteText, but that field was removed
-          `/json[type = ('Group', 'Person')]/referred_to_by[./classified_as/id='${getIRI(
-            'biographyStatement'
-          )}']/content`,
+          `/json[type = ('Group', 'Person')]/referred_to_by[./classified_as/equivalent/id='${IDENTIFIERS.biographyStatement}']/content`,
           // there is no field configuration for this one
-          `/json[type = ('Group', 'Person')]/referred_to_by[./classified_as/id='${getIRI(
-            'descriptionStatement'
-          )}' and ./language/id='${getLanguageIRI('en')}']/content`,
+          `/json[type = ('Group', 'Person')]/referred_to_by[./classified_as/equivalent/id='${IDENTIFIERS.descriptionStatement}' and ./language/equivalent/id='${IDENTIFIERS.langen}']/content`,
```

Comment:
@clarkepeterf, this is the only direct reference to a language identifier I found.  On purpose?

Reply:
A choice is to export LANGUAGE_EN from model.mjs and call getLanguageIdentifier.